Corona-Warn-App/src/main/java/de/rki/coronawarnapp/diagnosiskeys/download/KeyDownloadTool.kt

```diff
@@ -25,15 +23,7 @@ class KeyDownloadTool @Inject constructor(
         val keyInfo = cachedKey.info
 
         val preconditionHook: suspend (DownloadInfo) -> Boolean =
```

Comment:
Can remove the whole `preconditionHook`?

Reply:
removed it